jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/URIUtilTest.java

```diff
@@ -699,39 +747,24 @@ public void testFileUriGetUriLastPathSegment(String basePath, String expectedNam
         assertThat(URIUtil.getUriLastPathSegment(uri), is(expectedName));
     }
 
-    public static Stream<Arguments> uriLastSegmentSource() throws URISyntaxException, IOException
+    public static Stream<Arguments> uriLastSegmentSource() throws IOException
     {
         final String TEST_RESOURCE_JAR = "test-base-resource.jar";
+        List<Arguments> arguments = new ArrayList<>();
         Path testJar = MavenTestingUtils.getTestResourcePathFile(TEST_RESOURCE_JAR);
-        URI uri = new URI("jar", testJar.toUri().toASCIIString(), null);
+        URI jarFileUri = Resource.toJarFileUri(testJar.toUri());
```

Comment:
`Resource.toJarFileUri` looks useless now.

Reply:
Lets try to address this in the FileID/URIUtil second PR (not the first)